{
	"cve": {
		"data_type": "CVE",
		"data_format": "MITRE",
		"data_version": "4.0",
		"CVE_data_meta": {
			"ID": "CVE-2021-32694",
			"ASSIGNER": "security-advisories@github.com"
		},
		"problemtype": {
			"problemtype_data": [
				{
					"description": []
				}
			]
		},
		"references": {
			"reference_data": [
				{
					"url": "https://github.com/nextcloud/security-advisories/security/advisories/GHSA-h2gm-m374-99vc",
					"name": "https://github.com/nextcloud/security-advisories/security/advisories/GHSA-h2gm-m374-99vc",
					"refsource": "CONFIRM",
					"tags": []
				},
				{
					"url": "https://github.com/nextcloud/android/pull/7919",
					"name": "https://github.com/nextcloud/android/pull/7919",
					"refsource": "MISC",
					"tags": []
				},
				{
					"url": "https://hackerone.com/reports/859136",
					"name": "https://hackerone.com/reports/859136",
					"refsource": "MISC",
					"tags": []
				}
			]
		},
		"description": {
			"description_data": [
				{
					"lang": "en",
					"value": "Nextcloud Android app is the Android client for Nextcloud. In versions prior to 3.15.1, a malicious application on the same device is possible to crash the Nextcloud Android Client due to an uncaught exception. The vulnerability is patched in version 3.15.1."
				}
			]
		}
	},
	"configurations": {
		"CVE_data_version": "4.0",
		"nodes": []
	},
	"impact": {},
	"publishedDate": "2021-06-17T22:15Z",
	"lastModifiedDate": "2021-06-17T22:15Z"
}